cmd/jenkins-credentials-decryptor/main.go

```diff
@@ -45,3 +63,31 @@ func print(decryptedCredentials []xml.Credential) {
 		}
 	}
 }
+
+func printJson(decryptedCredentials []xml.Credential) {
```

Comment:
As this printing logic grew it would be nice to move it out of `main.go` to it's own file somewhere in the `pgk` folder.
Something like a `printer.go`.  
Along with `Credential` and  `print()`

That printer could accept the format it needs to print in (json or text).